src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/controller/UnallocatedCasesController.kt

```diff
@@ -6,28 +6,43 @@ import org.springframework.http.ResponseEntity
 import org.springframework.web.bind.annotation.GetMapping
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.hmppsallocations.jpa.entity.repository.UnallocatedCaseEntity
+import uk.gov.justice.digital.hmpps.hmppsallocations.jpa.repository.UnallocatedCasesRepository
+import java.time.LocalDateTime
 
 @RestController
 @RequestMapping(produces = [MediaType.APPLICATION_JSON_VALUE])
-class UnallocatedCasesController {
+class UnallocatedCasesController(
+  private val unallocatedCasesRepository: UnallocatedCasesRepository
+) {
 
   // TODO add role
   @GetMapping("/cases/unallocated")
-  fun unallocatedCases(): ResponseEntity<List<UnallocatedCase>> = ResponseEntity.ok(
-    listOf(
-      UnallocatedCase("Dylan Adam Armstrong", "J678910", "C1", "17 October 2021", "22 October 2021",	"Currently managed"),
-      UnallocatedCase("Andrei Edwards", "J680648", "A1", "18 October 2021", "23 October 2021", "New to probation"),
-      UnallocatedCase("Hannah Francis", "J680660", "C2", "20 October 2021", "", "Previously managed")
-
+  fun unallocatedCases(): ResponseEntity<List<UnallocatedCase>> {
+    return ResponseEntity.ok(
+      unallocatedCasesRepository.findAll().map {
```

Comment:
This one generate a technical debt that will need to be addressed later. If teh request is not filtered at production level will be equal to select *.